.changeset/big-gifts-wait.md

```diff
@@ -8,10 +8,12 @@
 
 ```diff
 +  --glide-core-color-effect-color-scroll-shadow: #0000001a;
++  --glide-core-color-interactive-surface-container-inactive--hover: #595959;
 ```
 
 ### Colors (Dark)
 
 ```diff
 +  --glide-core-color-effect-color-scroll-shadow: #ffffff0d;
++  --glide-core-color-interactive-surface-container-inactive--hover: #4f4f4f;
```

Comment:
Figured rather than a separate changeset, add on to this existing color changes in the hopper.  I left `Duration`s as a separate changeset, as it's a different "section".

Let me know if folks feel this isn't a good idea.